docs/installation/importing-css-assets-and-javascript.md

```diff
@@ -0,0 +1,162 @@
+# Import CSS, assets and JavaScript
+
+## CSS
+
+### Import all the CSS
+
+To import all the CSS styles from GOV.UK Frontend, add the following to your Sass file:
```

Comment:
Strictly speaking this imports SCSS which get compiled into CSS by the compiler. It might help to make this distinction so that user knows they'll be working with SCSS which they can override, extend etc. You can also import pure CSS files into SCSS.

Reply:
Ah ok - I've tweaked to "import all the SCSS rules" - does that sound ok?

Reply:
👍